lib/rubyplot/backend/magick_wrapper.rb

```diff
@@ -27,7 +33,8 @@ class MagickWrapper < Base
       PIXEL_MULTIPLIERS = {
         inch: 96,
         cm: 39.7953,
-        pixel: 1
+        pixel: 1,
+        point: 4/3
```

Comment:
Please add a comment here about what  exactly is point.

Reply:
Done